lib/game.rb

```diff
@@ -12,6 +12,7 @@ def initialize
     @computer_board = Board.new
     @computer_cruiser = Ship.new("Cruiser", 3)
     @computer_submarine = Ship.new("Submarine", 2)
+    @computer_array = [@player_board.cells.keys].flatten
```

Comment:
had to initialize this array because before we were recreating this array every time we did a .sample. This allowed us to store the shots which the computer had already fired upon and therefore not repeat them